content/en/docs/user-guides/how-to/notary-project-concepts.md

```diff
@@ -106,14 +106,21 @@ The path of the general configuration file of the `notation` CLI is as follows.
 ```
 
 ### Trust Store
+A trust store is a directory located within the filesystem that contains multiple collections of certificates, which are used to validate signatures. The Notary Project trust store currently support three types of certificates: 
+- Certificates from Certificate Authorities (CAs), which are stored in the `X509/ca` directory.
+- Signing Authority certificates, stored in the `X509/signingAuthority` directory. 
+- Time Stamping Authority (TSA) certificates, stored in the`X509/tsa` directory. 
```

Comment:
Time Stamping Authority (TSA) certificates are not supported yet. Could you add a note that this will be supported in the future release?

Reply:
I have updated the PR @FeynmanZhou. Please could you help review it again?

Reply:
Thanks for the updates!